pymilvus/client/grpc_handler.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, uri=config.GRPC_ADDRESS, host=None, port=None, channel=None,
         self._client_pem_path = kwargs.get("client_pem_path", "")
         self._client_key_path = kwargs.get("client_key_path", "")
         self._ca_pem_path = kwargs.get("ca_pem_path", "")
+        self._server_pem_path = kwargs.get("server_pem_path", "")
         self._server_name = kwargs.get("server_name", "")
```

Comment:
I wonder if we can just take these variables as temporary instead of saving them? Maybe it's not safe to save them.

Reply:
Just read from kwargs.get ?